ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Update.scala

```diff
@@ -71,17 +71,54 @@ object Update {
     * @param recordTime
     *   The ledger-provided timestamp at which the party was allocated.
     *
+    * @param submissionId
+    *   The submissionId of the command which requested party to be added.
+    *
     */
+  // TODO BH: add isLocal as was on async event
```

Comment:
The `isLocal` attribute can be figured out by the indexer itself (by comparing its own local participantId with the one provided in this message).

Reply:
addressed in latest changes for isLocal